lib/checks/language/valid-lang.tpl

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
The reason I left it there is so we could generate the list of languages again if we wanted to...but I also did not want the list of languages to have to be created with each build in case IANA moves that page. That is why I did not make the target part of the build target. Should I make it build every time?